testing/py3-pydocstyle/APKBUILD

```diff
@@ -0,0 +1,29 @@
+# Contributor: Russ Webber <russ@rw.id.au>
+# Maintainer: Russ Webber <russ@rw.id.au>
+pkgname="py3-pydocstyle"
+_pkgname="pydocstyle"
+pkgver="4.0.1"
+pkgrel=0
+pkgdesc="Static analysis tool for checking compliance with Python docstring conventions"
+url="https://pydocstyle.org/"
+arch="noarch"
+license="MIT"
+depends="python3 py3-snowballstemmer"
+makedepends="python3 py3-pytest py3-mock py3-six"
```

Comment:
depends will be installed during build too, so adding python3 to both makedepends and depends does nothing (just having it in depends is sufficient)

Reply:
This should most likely have py3-setuptools in its makedepends though